recipe/meta.yaml

```diff
@@ -8,79 +8,66 @@ source:
 build:
   preserve_egg_dir: False
   number: {{ GIT_DESCRIBE_NUMBER }}
-  missing_dso_whitelist:
-    - /lib64/libc.so.6            # [linux]
-    - /lib64/libm.so.6            # [linux]
-    - /lib64/libpthread.so.0      # [linux]
-    - /lib64/ld-linux-x86-64.so.2 # [linux]
 
 test:
   requires:
+    - python
+    - pip
     - python-wget
     - cvxpy                   # [linux]
     - scikit-image
-    - tomophantom=2.0.0       # [linux]
-    - tigre=2.6
+    - tomophantom 2.0.0       # [linux]
+    - tigre 2.6
     - packaging
-    - ccpi-regulariser=24.0.1 # [not osx]
-    - astra-toolbox=2.1=cuda* # [not osx]
-    - matplotlib-base >=3.3.0
+    - ccpi-regulariser 24.0.1 # [not osx]
+    - astra-toolbox 2.1 {{ environ.get('ASTRA_TOOLBOX_BUILDSTR', 'py*') }} # 'cuda*' not needed if GPU not present
+    - matplotlib-base >=3.3
     - zenodo_get >=1.6
 
   source_files:
     - ./Wrappers/Python/test
 
   commands:
     - pip install unittest-parametrize
-    - python -m unittest discover -v -s Wrappers/Python/test  # [win]
-    - python -m unittest discover -s Wrappers/Python/test -v  # [not win]
-
+    - python -m unittest discover -v -s Wrappers/Python/test
 
 requirements:
   build:
-    - python
+    - cmake >=3.16
+    - ninja
+    - {{ compiler('cxx') }}
```

Comment:
Does `conda_build_config` needed to specify `cxx_compiler` and `cxx_compiler_version` ?